### ISSUE TITLE
[Feature Request] Add option to select what a critical battery state

### ISSUE THREAD

**Issue by Null-Senpai** [Exalm/gnome-shell-extension-dynamic-battery#5]
Please add an option to set at what percentage the pure red icon shows.

**Comment by Exalm** [comment]
Yeah, the curent 10% threshold doesn't look well currently. There's a problem though, as this is normally synchronized with low battery notifications, and can be misleading if changed.
